sdk/keyvault/azure_security_keyvault_certificates/src/generated/clients/certificate_client.rs

```diff
@@ -351,8 +381,20 @@ impl CertificateClient {
         certificate_version: &str,
         options: Option<CertificateClientGetCertificateOptions<'_>>,
     ) -> Result<Response<Certificate>> {
+        if certificate_name.is_empty() {
+            return Err(azure_core::Error::message(
+                azure_core::error::ErrorKind::Other,
+                "parameter certificate_name cannot be empty",
+            ));
+        }
+        if certificate_version.is_empty() {
```

Comment:
This looks problematic as I assume that `certificate_version` should be optional.

I suspect that we need to update to the latest commit hash for the KV tsp which I believe has fixed this.